bin/pygrb/pycbc_make_offline_grb_workflow

```diff
@@ -252,16 +252,24 @@ if wflow.cp.has_option("workflow-segments", "segments-vetoes"):
 for ifo in ifos:
     avail_segs = select_segments_by_definer(veto_file.storage_path, ifo=ifo)
     avail_segs.coalesce()
```

Comment:
Please add a comment here to explain that avail_segs is empty if no relevant vetoes are found.